assembl/static2/js/app/components/debate/thread/answerForm.jsx

```diff
@@ -55,9 +59,22 @@ class AnswerForm extends React.PureComponent<*, AnswerFormProps, AnswerFormState
   };
 
   handleInputFocus = () => {
-    const isUserConnected = getConnectedUserId(); // TO DO put isUserConnected in the store
-    if (!isUserConnected) {
-      inviteUserToLogin();
+    const { debateData, identifier, handleAnswerClick } = this.props;
```

Comment:
Where is identifier coming from into the props?

Reply:
Yes